### DISCUSSION TITLE
关于应该忽略的不必要系统进程

### DISCUSSION THREAD

**Discussion by noberumotto** [Planshit/Tai#7] [General]
一些没必要记录使用时长的系统进程会写死在代码中，如果你发现了且方便提供的，可以回复在这条帖子下面，谢谢。

格式参考：

Windows版本：（7/10/11）
进程名（在Tai点进目标进程顶部信息可以右键复制）：xxx
进程文件路径：x:\xxx\xx.exe

**Comment by Youxikong** [comment]
Windows版本：Windows 10
进程名：Search application
进程文件路径：C:\WINDOWS\SystemApps\Microsoft.Windows.Search_cw5n1h2txyewy\SearchApp.exe

**Comment by tljk** [comment]
Windows版本：Windows 11
进程名：LockApp.exe
进程文件路径："C:\Windows\SystemApps\Microsoft.LockApp_cw5n1h2txyewy\LockApp.exe"
锁屏程序

**Comment by rebron1900** [comment]
可以自定义忽略列表么？

**Reply by noberumotto** [reply]
下一个版本会有这个自定义忽略列表功能

**Reply by noberumotto** [reply]
我是希望将一些系统进程都收集一下，在用户首次使用的时候不需要一个一个去处理

**Comment by rebron1900** [comment]
上次忘记给你提建议了，这次补上。

我觉得所有进程分类都应该交给用户处理，而且带分类功能，比如：

- 首次打开程序所有进程自动划入默认分类。
- 用户在使用一段时间后可以根据后台程序记录列表对程序分类，游戏的划到游戏一类，im的分类到im等等。（你上次说的统计游戏使用时长问题就可以这样解决了。）
- 还可以增加一个默认分类为不统计时长，其余用户不想统计的程序列入这个列表。

**Reply by noberumotto** [reply]
谢谢。分类交给用户处理是肯定的，我是想实现能识别判断出程序的类别在用户处理前先默认分成两大类（应用和游戏），这样能节省一些时间，对部分用户（包括我自己）来说挺方便的，也满足简单分类的需求

**Reply by rebron1900** [reply]
大家用的程序和玩的游戏都不一致，太繁杂了，做判断感觉要做很多数据收集。

**Reply by noberumotto** [reply]
我想能不能像安全软件一样通过收集游戏引擎的特征码来实现判断

**Reply by rebron1900** [reply]
我觉得没必要做这方面的工作，让用户进行分类我觉得是最科学的，而且自定义程度更高。
如果你说的这个特征码的形式实现起来不复杂的话也可以一起实现。
